frontend/src/store/uam/actions.ts

```diff
@@ -9,9 +10,20 @@ const getUsers = createAsyncThunk<
   AsyncThunkConfig
 >(ActionType.GET_USERS, async (_, { extra }) => {
   const { usersApi } = extra;
-  const usersDto = await usersApi.getAll();
+  const users = await usersApi.getAll();
 
-  return usersDto;
+  return users;
 });
 
-export { getUsers };
+const getPermissions = createAsyncThunk<
```

Comment:
you need to create new actions in the `frontend/src/store/uam-create-group/actions.ts`
actions should be per page